api-client/src/utils/getNavigationDeepLinks.ts

```diff
@@ -0,0 +1,29 @@
+import LocationModel from '../models/LocationModel'
+
+const getNavigationDeepLinks = (
```

Comment:
```suggestion
const getNavigationDeepLink = (
```
Optional and personal preference:
I think the term `navigation` is really misleading/ambivalent, I'd rather use something like
```suggestion
const getExternalMapsLink = (
```

Reply:
Please make sure that the naming is always the same, currently it sometimes is plural and sometimes singular (e.g. in the tests).